Scripts/Matlab/hopsan.m

```diff
@@ -0,0 +1,67 @@
+classdef hopsan
+    methods
+        function checkMessages(obj)
+            bufsize=1000;
+            buf=libpointer('string',blanks(bufsize));
+            status = 0;
+            while (status == 0)
+                [status,msg] = calllib('hopsanc','getMessage',buf,bufsize);    
+                disp(msg);
+            end
+        end
+        function obj = hopsan()
+            path = fullfile(getenv("HOPSANHOME"), '\bin\hopsanc.dll');
+            hpath = fullfile(getenv("HOPSANHOME"), '\hopsanc\include\hopsanc.h');
+            loadlibrary(path,hpath);
+            obj.checkMessages();
+        end
+
+        function setStartTime(obj,t)
+            calllib('hopsanc','setStartTime',t);
+            obj.checkMessages();
+        end
+        function setTimeStep(obj,t)
+            calllib('hopsanc','setTimeStep',t);
+            obj.checkMessages();
+        end
+        function setStopTime(obj,t)
+            calllib('hopsanc','setStopTime',t);
+            obj.checkMessages();
+        end
+        function setLogSamples(obj,t)
+            calllib('hopsanc','setLogSamples',t);
+            obj.checkMessages();
+        end
+        function loadModel(obj,path)
+            calllib('hopsanc','loadModel',path);
+            obj.checkMessages();
+        end
+        function loadLibrary(obj,path)
+            calllib('hopsanc','loadLibrary',path);
+            obj.checkMessages();
+        end
+        function simulate(obj)
+            calllib('hopsanc','simulate');
+            obj.checkMessages();
+        end
+        function t = getTimeVector(obj)
+            nSamples = calllib('hopsanc','getLogSamples');
```

Comment:
getLogSamples and setLogSamples need update due to changes in the other branch. 
"NumberOf"

Reply:
Fixed. But now this PR will not work in the current state, since the "NumberOf" fix is in the Python PR, which is not yet merged with master. Not a big problem though.